### PR TITLE
PostList Component

### PR DESCRIPTION
**Pull request by a36168120**
- add all needed content for displaying 
- basic styling setup

(Sprint-2)TODO: 
- add mutation for createPost & createComment 
- add function in PostButton popup 
  * add FavesPost
  * copy post link
  * delete post

### CODE REVIEW

File: client/js/components/PostList/PostList.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
...HDyKgOBq4fTa4fjpMd9B/VTrhqU10SfZVviRUbTsmsAeS+sxjQwSZ+aegGnPRVPHn1jRDOzaDAAa2pq6Bzp5cs+RVoxy97Wq1u7WuMDqW7n+bL7KrcSVSxpMwdqjm6ubm0FGl1qO2nlqkDJH4dYVFtmA+Z7oEglrRoG6bQZ0VkdSIJa4e6gPh1etFJ9PQPa/M4AyIIDRl6gZcs84nmr6xgqCSJCKGmM8JsW29PuCA4lxHnzHsvPHxCu+1xG5d/1C0eTAG/ovTFxlYwnkB9gvJuJ3Ha1alT89R7v5nE/qn0JjMlcCilcokQXvgJINnUpUoEAFyri1DKAlAAQgXFBKADI0oqFABpXBFBQoAOuBRUMoAPK4osoZQB6I+FVfNa2p/6OT1ZA/8AFWrGq3Zsc4bkQPMrP/gpdh1pTZzZWqt95f8A+StvGlQ5WtHOSeumgA8TKa6JFQtbpz6j3g5Wt7jCeQHzP8T/AECi+IHaCDlhpLSf+VT/ABV3dah2aPFPmDLAAnXRv4Z5F3gN/RVriWvnJptcS2c1R3N7hsT4DkPVCBiXC+LdhcioBDDDS3pT2APiBB8wtywCpmpu1nXTyiV51t3w5bf8ObztLctJ1aY9I0/p6IT2Hof8bXvYWNxU/LRfHmWkD6leWXaCF6B+NF72eHuZOtSoxnpOY/RpXn2qU2IIVyKpHh+2p1rqjRqfLUfk3jvOBDNeXeLVEQxQStdt+DsNpD97Sa4g/MatWI8ZcB9FmPEdvTpXVZlItNMVDkykEZXd4AEdJj0UVNN0WzxSjFSfsjpXEoriuUio4osoSioAUQooK4FABpQoqEFABkMoqGUAGQtKKuBQBqvwPvIe+l0qsf8Azsc0/wDwCv8AxVcZqxH5WgHwmTA8Tp7LHfhRe9niDG8n5R6h7Y+hK0Xi/ExRqVHbuL8rG8pgDfl8v08oaJERjF4WkUWa1H7/AMLeZJ6fePeAxSjkaGjXmTzJ6qVtrY05e8zUfq8/Zo6AKLxt+mvXRRb9DohZ1WofC65LXkE6PaQ0dXM1JHgAfqsysrc1XhoMSTr0AEudHOBy8leuFroU69FzRAMZR+WhOWfFz3GShCsS+PV73reh/rqH0hrfu5Y9UKv3xovc+JuZypUqbPUgvP0ePZZ84qTIghCDGoMEGQRoQUVckBxaECElFcUAAEYoAuKAAKBcSgQB/9k=',
```
Comment (siddharthparmar7) [inline client/js/components/PostList/PostList.js]: why this bog string here?

File: client/js/components/PostList/styles.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+  commentBtn: {
+    flexDirection: 'row',
+    width: '100%',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+});
+
+export default styles;
```
Comment (siddharthparmar7) [inline client/js/components/PostList/styles.js:85]: use %s for padding, margin, height, width.

File: client/js/lib/helper.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+  let options = {
+    month: 'short',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+  };
+
+  return date.toLocaleDateString('en-us', options).replace(',', ' at');
+};
```
Comment (siddharthparmar7) [inline client/js/lib/helper.js:12]: use moment.js.

File: client/js/navigation/AppStackNavigation.js [hunk 4]
```diff
@@ -4,8 +4,16 @@ import CommunityScreen from '../screens/Community';
...
 
-const CommunityStack = createStackNavigator({Community: CommunityScreen});
+const CommunityStack = createStackNavigator({
+  Community: {
+    screen: CommunityScreen,
+  },
+  // Comments: {
+  //   screen: CommentsScreen,
+  // },
+});
```
Comment (siddharthparmar7) [inline client/js/navigation/AppStackNavigation.js:18]: remove commented code.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work! Please make the requested changes. Thank you.

File: client/js/lib/helper.js (hunk 3 above)
Comment (oliwright1994) [reply client/js/lib/helper.js:12]: We considered it but this is the only place we're formatting a date and we think there's no point installing an entire package (and a fairly large one as well) for one function, especially given we have a solution already.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please resolve merge conflicts.

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**